docs/Materials.md.html

```diff
@@ -2172,6 +2172,7 @@
 **getWorldGeometricNormalVector()**     | float3   |  Normalized normal in world space, before bump mapping (can be used before `prepareMaterial()`)
 **getWorldReflectedVector()**           | float3   |  Reflection of the view vector about the normal (must be used after `prepareMaterial()`)
 **getNormalizedViewportCoord()**        | float3   |  Normalized viewport position (i.e. NDC coordinates normalized to [0, 1], can be used before `prepareMaterial()`)
+**getObjectAabbCenterWorldSpace()**     | float3   |  Position of the object's axis-aligned bounding box's center in world space (see note below about world-space) 
```

Comment:
Its name is now `getMaterialOrientationCenter()`. Also `getMaterialOrientationMatrix()` is missing.

Reply:
Fixed in 86c38a3